concepts/concept_management.ipynb

```diff
@@ -0,0 +1,338 @@
+   "outputs": [],
+   "source": [
+    "app.create_concepts(concept_ids=[\"cat\", \"dog\", \"animal\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Add Concept Relation\n",
+    "\n",
+    "To add concept relations, we have to provide the `subject concept id`, a list of `object concept ids` with which we need to create a list of corresponding `concept relations`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.create_concept_relations(\"cat\", [\"animal\", \"dog\"], [\"hyponym\", \"synonym\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Search Concept Relation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get all the concept relations of the app, we should directly call the **App** class **search_concept_relations** function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.search_concept_relations()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can use the **concept_id** and **predicate** arguments of **search_concept_relations** as filters to get the required concept relations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.search_concept_relations(concept_id=\"cat\",predicate=\"synonym\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To display the concept relations in a rich Tree structure, we have add True to **show_tree** argument of **search_concept_relations**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.search_concept_relations(show_tree=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Delete Concept Relation\n",
+    "\n",
+    "To delete the concept relation we require the `concept id` and `concept relation ids`. But if we have to delete the all the concept relations, we can just provide the `concept id`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "app.delete_concept_relations(\"cat\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Concept based Input Search\n",
+    "\n",
+    "To search for Inputs related to specific concepts we use search filters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "search = app.search(top_k=2, metric=\"euclidean\")\n",
+    "search.query(filters=[{\n",
+    "    \"concepts\": [{\n",
+    "        \"name\": \"dog\",\n",
+    "        \"value\": 1\n",
```

Comment:
Not particularly related to this ticket, but can we search with just name? If thats possible, we should add examples without value(maybe one with value) since its not used much in the platform.

Reply:
Initially tried with just name, but it didn't work